test/unit_tests/utils/process-json-spec.js

```diff
@@ -92,163 +103,202 @@ describe( 'process-json', () => {
 
   } );
 
   describe( 'processNumOriginationsData', () => {
-    const data = {
-      test: {
-        adjusted: [
-          [ 1117584000000, 1 ],
-          [ 1230768000000, 1239123 ],
-          [ 1233446400000, 888888 ],
-          [ 1235865600000, 1231125 ],
-          [ 1238544000000, 82364821 ],
-          [ 1241136000000, 7654321 ],
-          [ 1243814400000, 1234567 ],
-          [ 1246406400000, 1212123 ],
-          [ 1249084800000, 3434343 ]
-        ],
-        unadjusted: [
-          [ 1117584000000, 1 ],
-          [ 1230768000000, 1239123 ],
-          [ 1233446400000, 888888 ],
-          [ 1235865600000, 1231125 ],
-          [ 1238544000000, 82364821 ],
-          [ 1241136000000, 7654321 ],
-          [ 1243814400000, 1234567 ],
-          [ 1246406400000, 1212123 ],
-          [ 1249084800000, 3434343 ]
-        ]
-      }
-    };
-    const test = originations( data, 'test' );
+    beforeEach( () => {
+      data = {
+        test: {
+          adjusted: [
+            [ 1117584000000, 1 ],
+            [ 1230768000000, 1239123 ],
+            [ 1233446400000, 888888 ],
+            [ 1235865600000, 1231125 ],
+            [ 1238544000000, 82364821 ],
+            [ 1241136000000, 7654321 ],
+            [ 1243814400000, 1234567 ],
+            [ 1246406400000, 1212123 ],
+            [ 1249084800000, 3434343 ]
+          ],
+          unadjusted: [
+            [ 1117584000000, 1 ],
+            [ 1230768000000, 1239123 ],
+            [ 1233446400000, 888888 ],
+            [ 1235865600000, 1231125 ],
+            [ 1238544000000, 82364821 ],
+            [ 1241136000000, 7654321 ],
+            [ 1243814400000, 1234567 ],
+            [ 1246406400000, 1212123 ],
+            [ 1249084800000, 3434343 ]
+          ]
+        }
+      };
+      testData = originations( data, 'test', 'inquiry_test_file.csv' );
+    } );
 
     it( 'should eliminate dates before 2009', () => {
-      expect( test.adjusted[0][0] ).toBe( 1230768000000 );
+      expect( testData.adjusted[0][0] ).toBe( 1230768000000 );
     } );
 
     it( 'should identify and select the proper group', () => {
-      expect( test.adjusted[0][0] ).toBe( 1230768000000 );
+      expect( testData.adjusted[0][0] ).toBe( 1230768000000 );
     } );
 
-    it( 'should assign the correct projected Dates', () => {
-      expect( test.projectedDate.timestamp ).toBe( 1235865600000 );
-      expect( test.projectedDate.label ).toBe( 'February 2009' );
+    it( 'should assign the correct projected dates, ' +
+        '4 months for inquiry index charts', () => {
+      expect( testData.projectedDate.timestamp ).toBe( 1241136000000 );
+      expect( testData.projectedDate.label ).toBe( 'April 2009' );
     } );
 
+    it( 'should assign the correct projected dates, ' +
+        '0 months for inquiry index charts', () => {
+      const testDataDen = originations( data, 'test', 'denials_test_file.csv' );
+      expect( testDataDen.projectedDate.timestamp ).toBeNull();
```

Comment:
@cfarm Is this and the line below the expected result for the denial data?

Reply:
It's been updated to undefined so that it passes the default value to highcharts.